_ml-commons-plugin/api/model-apis/batch-predict.md

```diff
@@ -31,7 +31,7 @@ POST /_plugins/_ml/models/<model_id>/_batch_predict
 
 ## Prerequisites
 
-Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request:
+Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The optional `action_type` paramter supports canceling the batch job running on OpenAI:
```

Comment:
The optional action_type parameter allows for checking the status or canceling a batch job running on OpenAI.

Reply:
Can we have something like this: 
Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The action_type `cancel_batch_predict` is optional and it supports canceling the batch job running on OpenAI: